ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -40,10 +40,6 @@ tests:
     env:
       TEST_REQUIRES_SSH: "yes"
       TEST_TYPE: upgrade-conformance
-    pre:
-    - ref: ipi-install-hosted-loki
-    - ref: tcpdump-service
-    - chain: ipi-azure-pre-stableinitial
```

Comment:
why is the install gone?

Reply:
see above. note that both of these were added when the change went in originally